app/lib/publisher/backend.dart

```diff
@@ -169,6 +206,41 @@ class PublisherBackend {
     });
   }
 
+  /// A callback from consent backend, when a consent is granted.
+  Future inviteConsentGranted(String publisherId, String userId) async {
+    await _db.withTransaction((tx) async {
+      final key = _db.emptyKey
+          .append(Publisher, id: publisherId)
+          .append(PublisherMember, id: userId);
+      final list = await tx.lookup<PublisherMember>([key]);
+      final member = list.single;
+      if (member == null) {
+        throw NotFoundException('Membership invite was deleted.');
+      }
+      if (member.isPending) {
+        member.isPending = false;
+        tx.queueMutations(inserts: [member]);
+        await tx.commit();
```

Comment:
Handle exceptions... we need to check if there was a competing request that this exactly this, in which case we can return ok, since the API is then idempotent.